Customer request to highlight the order they need to upgrade in when … (#2436)
elastic/observability-docs: docs/en/ingest-management/fleet/air-gapped.asciidoc

content.
* {agent}s are able to download binaries during upgrades.

> When upgrading all the components in the Air-gapped environment it is recommended that they are upgraded in the following order:

Reviewer: ```suggestion
When upgrading all the components in an air-gapped environment, it is recommended that you upgrade in the following order:
```

> 1) Upgrade the EPR
> 2) Upgrade the {stack} including {kib}
> 3) Upgrade the {artifact-registry} and ensure the latest {agent} binaries are available
> 3) Upgrade the on-premise {fleet server}
> 4) Lastly issue an upgrade from {fleet} for all the {agent}s

Reviewer: ```suggestion
. Upgrade the {package-registry}.
. Upgrade the {stack} including {kib}.
. Upgrade the {artifact-registry} and ensure the latest {agent} binaries are available.
. Upgrade the on-premise {fleet-server}.
. In {fleet}, issue an upgrade for all the {agent}s.
```
